app/js/components/orderEntry/OrderEntryPage.jsx

```diff
@@ -300,34 +316,29 @@ export class OrderEntryPage extends PureComponent {
               note={this.props.note}
               location={this.props.location}
             />
-            <div className="header-nav">
-              <div>
-                <h3
-                  className="orders-nav"
-                  onClick={() => this.switchOrderType()}
-                  role="button">
-                  <b>
-                    <FormattedMessage
-                      id="app.orders.list"
-                      defaultMessage="Orders List"
-                      description="Orders List" />
-                  </b>
-                </h3>
-              </div>
-              {<SelectOrderType
-                switchOrderType={this.switchOrderType}
-                currentOrderType={this.props.currentOrderType}
-                page={page}
-              />}
-            </div>
-            <div className="body-wrapper drug-order-entry">
-              <RenderOrderType
-                backLink={returnUrl}
-                currentOrderTypeID={this.props.currentOrderType.id}
-                {...this.props}
-              />
-              {this.props.currentOrderType.id && this.renderDraftOrder()}
-            </div>
+            <Tabs
+              selectedIndex={this.state.index} onSelect={ (i) => {
+                this.switchTabs(i); this.setState({ index: i });
+              }
+              }>
+              <TabList>
+                <Tab><FormattedMessage id="app.orders.list" defaultMessage="Orders List" description="Orders List" /></Tab>
+                <Tab><FormattedMessage id="app.orders.labs.add" defaultMessage="Add Lab Orders" description="Add Lab Orders" /></Tab>
```

Comment:
Sorry, this wasn't clear from the ticket, but there needs to be an option for "Drug Orders" as well, and then we determine which tabs are available (Drug Orders, Test Orders, or both) based on a passed-in "page" attribute (see attached screenshot)... if page=laborders only show laborders tab, if page=testorders, only show Test Orders tab, if page attribute not present, show both.

![2019-10-04_11-33](https://user-images.githubusercontent.com/1163015/66220374-e69e5900-e69a-11e9-923a-4904b578a9af.png)

I don't remember exactly how it currently is achieved, but it looks like the logic to handle this in is SelectOrderType.jsx.